lib/wallaby/browser.ex

```diff
@@ -1015,16 +1013,16 @@ defmodule Wallaby.Browser do
   @doc """
   Accepts all subsequent JavaScript dialogs in the given session.
   """
-  def accept_dialogs(%Session{}=session) do
-    Driver.accept_dialogs(session)
+  def accept_dialogs(%Session{driver: driver}=session) do
```

Comment:
I completely agree on credo. I'm just haven't been ready to fix all of the style issues throughout the code-base at the moment.